docs/amber_script.md

```diff
@@ -75,31 +80,33 @@ TODO(dneto): Support half-precision floating point.
 Sized arrays and structures are not currently representable.
 
 ```
-// Filling the buffer with a given set of data. The values must provide
-// <size_in_bytes> of <type> data. The data can be provided as the type or
-// as a hex value.
-
+// Filling the buffer with a given set of data. The values must be
+// of <type> data. The data can be provided as the type or as a hex value.
 BUFFER <name> DATA_TYPE <type> DATA
 <value>+
 END
 
+// Defines a buffer which is filled with data as specified by the `initializer`.
 BUFFER <name> DATA_TYPE <type> SIZE <size_in_items> <initializer>
+
+// Creates a buffer which will store the given `FORMAT` of data. These
+// buffers are used as image and depth buffers in the `PIPELINE` commands.
+// The buffer will be sized based on the `RENDER_SIZE` of the `PIPELINE`.
+BUFFER <name> FORMAT <format_string>
```

Comment:
Thoughts on using the <format_string> which is what vKScript takes, or changing DATA_TYPE to allow more complex things? Need to be able to represent things like D24_FLOAT_S8_UINT nicely, and R8G8B8A8_UINT_PACK32.